### ISSUE TITLE
[Request]: Squad Spawner: Option to FORCE run over.

### ISSUE THREAD

**Issue by MickyMous999** [Yimura/YimMenu#1711]
### Problem

So i've been messing with this feature, pretty fun, but i realized that the peds won't run you over in a BMX, or when they got a gun, also when i try to spawn them with NO guns, they still spawn with an UZI idk why.
This option would make the rc cars even funnier

### Solution

Add an option in "advanced settings" to force peds to run players over no matter what, and avoid shooting if possible.

### Reason

Because it would be a fun feature to mess with my friends or against annoying modders with god mode 👍

**Comment by Tayipc38** [comment]
"Stay In Vehicle" toggleable should make the driver run over the target. As for the smg, it seems some debug lines made it to the final cut.

**Comment by MickyMous999** [comment]
Yeah i had that option toggled, they will only run over when they got no bullets, but it seems like they have infinite bullets with that smg idk
